core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -629,10 +629,8 @@ private StorageDirView allocateSpace(long sessionId, AllocateOptions options)
           // Block expansion are forcing the location. We do not want the review's opinion.
           dirView = mAllocator.allocateBlockWithView(sessionId, options.getSize(),
               options.getLocation(), allocatorView.refreshView(), true);
-          if (LOG.isDebugEnabled()) {
-            LOG.debug("Allocation after freeing space for block expansion. availableBytes: {}",
-                dirView.getAvailableBytes());
-          }
+          LOG.debug("Allocation after freeing space for block expansion, {}", dirView == null
```

Comment:
there is actually a reason to do this check before the debug statement. because it saves the cost of computing parameters to the debug statement. in this case `dirView.getAvailableBytes` is not called.